code/modules/mob/living/simple_animal/abnormality/teth/faelantern.dm

```diff
@@ -0,0 +1,369 @@
+#define STATUS_EFFECT_FAIRYLURE /datum/status_effect/fairy_lure
+/mob/living/simple_animal/hostile/abnormality/faelantern
+	name = "Faelantern"
+	desc = "A small fairy with a green glow sits atop a delicate tree branch."
+	icon = 'ModularTegustation/Teguicons/64x64.dmi'
+	icon_state = "faelantern"
+	icon_living = "faelantern_fairy"
+	icon_dead = "faelantern_egg"
+	maxHealth = 1200
+	health = 1200
+	base_pixel_x = -16
+	pixel_x = -16
+	threat_level = TETH_LEVEL
+	work_chances = list(
+		ABNORMALITY_WORK_INSTINCT = 30,
+		ABNORMALITY_WORK_INSIGHT = list(45, 45, 50, 55, 55),
+		ABNORMALITY_WORK_ATTACHMENT = 45,
+		ABNORMALITY_WORK_REPRESSION = 60
+			)
+	damage_coeff = list(BRUTE = 1, RED_DAMAGE = 1.3, WHITE_DAMAGE = 0.5, BLACK_DAMAGE = 1.5, PALE_DAMAGE = 2)
+	can_patrol = FALSE
+	can_breach = TRUE
+	del_on_death = FALSE
+	deathmessage = "creaks and crumbles into its core."
+	ranged = TRUE
+
+	work_damage_amount = 5
+	work_damage_type = WHITE_DAMAGE
+	start_qliphoth = 1
+	max_boxes = 12
+	ego_list = list(
+		/datum/ego_datum/weapon/faelantern,
+		/datum/ego_datum/armor/faelantern
+		)
+
+	gift_type = /datum/ego_gifts/faelantern
+	gift_message = "The fairy extends an olive branch towards you."
+
+	abnormality_origin = ABNORMALITY_ORIGIN_LIMBUS
+	var/can_act = FALSE
+	var/break_threshold = 450
+	var/broken = FALSE
+	var/fairy_enabled
+	var/fairy_health = 1200
+	var/lure_cooldown
+	var/lure_cooldown_time = 30 SECONDS
+	var/lure_damage = 20
+	var/stab_cooldown
+	var/stab_cooldown_time = 30
+	var/lured_list = list()
+
+/mob/living/simple_animal/hostile/abnormality/faelantern/AttackingTarget(atom/attacked_target)
+	return OpenFire()
+
+/mob/living/simple_animal/hostile/abnormality/faelantern/Goto(target, delay, minimum_distance)
+	return FALSE
+
+/mob/living/simple_animal/hostile/abnormality/faelantern/Move()
+	return FALSE
+
+/mob/living/simple_animal/hostile/abnormality/faelantern/death(gibbed)
+	. = ..()
+	for(var/mob/living/carbon/human/H in lured_list)
+		EndEnchant(H)
+	icon_state = icon_dead
+	playsound(src, 'sound/abnormalities/doomsdaycalendar/Limbus_Dead_Generic.ogg', 100, 1)
+	animate(src, alpha = 0, time = 10 SECONDS)
+	QDEL_IN(src, 10 SECONDS)
+
+/mob/living/simple_animal/hostile/abnormality/faelantern/PostWorkEffect(mob/living/carbon/human/user, work_type, pe, work_time, canceled)
+	if(work_type == ABNORMALITY_WORK_REPRESSION && work_time < 16 SECONDS)
```

Comment:
Do not use work time, check temperance bonus instead. Work time is inconsistant due to server ticks.